.travis.yml

```diff
@@ -1,7 +1,29 @@
+sudo: false
+
 language: d
 
-#compiler:
-#  - dmd
+os:
+ - linux
+ - osx
```

Comment:
The OS X testers aren't enabled by default, but if you send a quick email to support@travis-ci.com they will be (hopefully) happy to enable them for you. I have had no problem getting them for my projects.